docs/content/en/lazy-load-translations.md

```diff
@@ -46,15 +46,17 @@ Configuration example:
   ],
   lazy: true,
   langDir: 'lang/',
-  defaultLocale: 'en'
+  vueI18n: {
+    defaultLocale: 'en'
+  }
```

Comment:
This is not correct. `vue-i18n` doesn't support a `defaultLocale` option.

Reply:
You're right. Sorry. This was meant to be a reference to `fallbackLocale`. I think that without it, it wasn't working for when using lazy-loading.

Reply:
Actually, I think it was working fine.  It seems that something else was at play, as some kind of caching possibly lead me to believe this was the critical change, but I've left the changes for the missing return statement.